delivery_carrier_label_batch/wizard/generate_labels.py

```diff
@@ -139,18 +139,22 @@ def _get_all_files(self, batch):
                 groups.setdefault(picking.id, []).append((pack, picking, label))
 
         for group in groups.values():
```

Comment:
This is just a minor non-blocking suggestion.
```suggestion
        is_in_testing = getattr(threading.current_thread(), 'testing', False)
        for group in groups.values():
```